private/bufpkg/bufimage/bufimagemodify/bufimagemodifyv2/bufimagemodifyv2.go

```diff
@@ -165,6 +172,67 @@ func ModifyJavaPackage(
 	return nil
 }
 
+// TODO: Decide whether or not to use this one (this isn't the function we are calling right now,
+// but it's an alternative way of modifying). The interface is not as nice as that of FieldOptionModifier,
+// but this mean less code.
+// ModifyJSType modifies JS_TYPE field option.
+func ModifyJSType(
+	imageFile bufimage.ImageFile,
```

Comment:
We could use this as well, instead of instantiating a field options modifier and then calling its method `ModifyJSType` (see the interface below).
This function has a slightly less straightforward signature, but it contains less code comparing to the code for creating a field options modifier + the code for that method on field options modifier.